.github/pull_request_template.md

```diff
@@ -1,27 +1,21 @@
 #### Pull Request Checklist
 
+- [ ] All changes are Go version 1.15 compliant
+- [ ] Your PR passes all CI checks
 - [ ] If this is your first time contributing, we recommend you read the [Code
   Contribution Guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md)
-- [ ] All changes are Go version 1.12 compliant
-- [ ] The code being submitted is commented according to [Code Documentation and Commenting](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#CodeDocumentation)
+   - [ ] The code being submitted is commented according to [Code Documentation and
+  Commenting](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#CodeDocumentation)
+   - [ ] Commits have a logical structure according to [Ideal Git Commit
+  Structure](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#IdealGitCommitStructure)
 - [ ] For new code: Code is accompanied by tests which exercise both
   the positive and negative (error paths) conditions (if applicable)
-- [ ] For bug fixes: Code is accompanied by new tests which trigger
+- [ ] For bug fixes: If possible code is accompanied by new tests which trigger
```

Comment:
nitnit: "If possible, code"

Reply:
Ah, wasn't sure here. Never got the hang of all the English comma rules :)

Reply:
I _thiiink_ it's right, but tbh me too.